File: components/physics/src/main/java/org/storm/physics/math/geometry/shapes/Triangle.java
package org.storm.physics.math.geometry.shapes;

import org.storm.physics.math.geometry.Point;

/**
 * A Triangle is a ConvexPolygon with three sides and is the smallest possible ConvexPolygon.
 */
public class Triangle extends ConvexPolygon {

  public Triangle(Point a, Point b, Point c) {
    super(a, b, c);
  }

}

File: components/physics/src/main/kotlin/org/storm/physics/math/geometry/shapes/Triangle.kt
package org.storm.physics.math.geometry.shapes

import org.storm.core.graphics.geometry.Point

/**
 * A Triangle is a ConvexPolygon with three sides and is the smallest possible ConvexPolygon.
 */
open class Triangle(a: Point, b: Point, c: Point) : Polygon(a, b, c)
